src/watchmaker/__init__.py

```diff
@@ -224,6 +224,10 @@ def __init__(self, arguments):
 
         self.log.debug('System Parameters: %s', self.system_params)
 
+        # Make 'admin_groups' lowercase on Linux systems
```

Comment:
I think it would be better to handle this logic in `workers/salt.py`, since this argument is specific to that worker.

What we can do is delete this line:

* https://github.com/plus3it/watchmaker/blob/9805a27f684ab43e1bd7fd03799dab4c9d52f727/src/watchmaker/workers/salt.py#L96

And insert the same line into the SaltLinux and SaltWindows init functions.

* https://github.com/plus3it/watchmaker/blob/9805a27f684ab43e1bd7fd03799dab4c9d52f727/src/watchmaker/workers/salt.py#L520
* https://github.com/plus3it/watchmaker/blob/9805a27f684ab43e1bd7fd03799dab4c9d52f727/src/watchmaker/workers/salt.py#L669

And only in the SaltLinux init, force the value to lowercase.